modules/cellranger/Dockerfile

```diff
@@ -4,7 +4,7 @@ LABEL authors="Gisela Gabernet <gisela.gabernet@gmail.com>" \
 # Disclaimer: this container is not provided nor supported by 10x Genomics.
 
 # Install procps and clean apt cache
```

Comment:
We need to decide how we are going to use these `Dockerfile` properly for these scenarios.

Reply:
On my TODO list to auto build and push any! Although this one is going to need to be ignored because of the 10x download :grimacing: